.github/workflows/azure-vm.yml

```diff
@@ -0,0 +1,52 @@
+name: Deploy to an Azure Virtual Machine
+
+env:
+  OUTPUT_PATH: ${{ github.workspace }}
+  DOTNET_VERSION: '6.0.x'
```

Comment:
The dotnet version should also be noted in the README. 7.0 just came out - it shouldn't cause a problem, but if it did it would be good to have that hint.

Reply:
I believe that the dotnet version is already noted in the README.